### PR TITLE
Fix API Docs configuration

### LINKED ISSUE #149: Fix API scala documentation.
Scaldoc needs to be configured to be generated and linked with the website.

### PR DESCRIPTION
**Pull request by Avasil**
Fixes #149 but it's missing Parquet - there were some errors related to ProtoBuf, you can see them if you add parquet module and run `sbt docs/docusaurusCreateSite`

### CODE REVIEW

File: parquet/src/main/scala/monix/connect/parquet/ParquetPublisher.scala [hunk 1]
```diff
@@ -35,18 +35,20 @@ private[parquet] class ParquetPublisher[T](reader: ParquetReader[T]) {
     */
   private def readRecords(sub: Subscriber[T]): Task[Unit] = {
     val t = Task(reader.read())
-    t.onErrorHandleWith { case ex: Throwable => Task(sub.onError(ex)) }
-    t.flatMap { r =>
-      if (r != null) {
-        Task.deferFuture(sub.onNext(r)).flatMap {
-          case Ack.Continue => readRecords(sub)
-          case Ack.Stop => Task.unit
+    t.redeemWith(
```
Comment (Avasil) [inline parquet/src/main/scala/monix/connect/parquet/ParquetPublisher.scala:38]: I noticed there was a warning, earlier code was discarding `t.onErrorHandleWith(...)` Task

**Comment by paualarco** [comment]
> Fixes #149 but it's missing Parquet - there were some errors related to ProtoBuf, you can see them if you add parquet module and run `sbt docs/docusaurusCreateSite`

Thanks @Avasil :)
It's kind of weird that a package from the target folder is causing the build to fail... let's merge these changes and update the ticket #149

**Comment by Avasil** [comment]
Maybe it's possible to exclude it or something, I didn't look at it :D

**Comment by Avasil** [comment]
e.g. similarly to
```
  // Exclude monix.*.internal from ScalaDoc
  sources in (ScalaUnidoc, unidoc) ~= (_ filterNot { file =>
    // Exclude all internal Java files from documentation
    file.getCanonicalPath matches "^.*monix.+?internal.*?\\.java$"
  }),
```

**Comment by paualarco** [comment]
yeah that's what I was thinking too, ideally it should not cause any problems xD
created this one for the moment https://github.com/monix/monix-connect/issues/193

**Pull request merged**